src/video/cocoa/cocoa_v.mm

```diff
@@ -638,38 +457,10 @@ - (void)drawRect:(NSRect)invalidRect
 	}
 }
 
-
-WindowQuartzSubdriver::WindowQuartzSubdriver()
-{
-	this->window_width  = 0;
-	this->window_height = 0;
-	this->buffer_depth  = 0;
-	this->window_buffer  = NULL;
-	this->pixel_buffer  = NULL;
-	this->active        = false;
-	this->setup         = false;
-
-	this->window = nil;
-	this->cocoaview = nil;
-
-	this->cgcontext = NULL;
-
-	this->num_dirty_rects = MAX_DIRTY_RECTS;
-}
-
-WindowQuartzSubdriver::~WindowQuartzSubdriver()
-{
-	/* Release window mode resources */
-	if (this->window != nil) [ this->window close ];
-
-	CGContextRelease(this->cgcontext);
-
-	CGColorSpaceRelease(this->color_space);
-	free(this->window_buffer);
-	free(this->pixel_buffer);
-}
-
-void WindowQuartzSubdriver::Draw(bool force_update)
+/** Draw window
```

Comment:
missing newline

Reply:
The Doxygen comments were all over the place in the code, I probably missed a lot that could be reformatted.